jobconfig.toml

```diff
@@ -175,14 +175,14 @@
 [LAWYER]
 "Playtime Requirements" = 0
 "Required Account Age" = 0
-"Spawn Positions" = 2
-"Total Positions" = 2
+"Spawn Positions" = 1
+"Total Positions" = 1
```

Comment:
We can probably keep these at 2. Civil disputes need 2 lawyers don't they

Reply:
I suppose so, let me make it so there can be two